cirras-underwriting-api/cirras-underwriting-persistence/src/test/java/ca/bc/gov/mal/cirras/underwriting/persistence/v1/dao/VerifiedYieldContractCommodityDaoTest.java

```diff
@@ -0,0 +1,348 @@
+package ca.bc.gov.mal.cirras.underwriting.persistence.v1.dao;
+
+import java.time.LocalDate;
+import java.time.ZoneId;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+import org.springframework.test.context.junit4.SpringJUnit4ClassRunner;
+
+import ca.bc.gov.mal.cirras.underwriting.persistence.v1.dto.GrowerContractYearDto;
+import ca.bc.gov.mal.cirras.underwriting.persistence.v1.dto.InventoryContractCommodityDto;
+import ca.bc.gov.mal.cirras.underwriting.persistence.v1.dto.DeclaredYieldContractDto;
+import ca.bc.gov.mal.cirras.underwriting.persistence.v1.dto.InventoryContractDto;
+import ca.bc.gov.mal.cirras.underwriting.persistence.v1.dto.VerifiedYieldContractCommodityDto;
+import ca.bc.gov.mal.cirras.underwriting.persistence.v1.dto.VerifiedYieldContractDto;
+import ca.bc.gov.mal.cirras.underwriting.persistence.v1.spring.PersistenceSpringConfig;
+import ca.bc.gov.nrs.wfone.common.persistence.dao.DaoException;
+import ca.bc.gov.nrs.wfone.common.persistence.dao.NotFoundDaoException;
+
+
+@RunWith(SpringJUnit4ClassRunner.class)
+@ContextConfiguration(classes= {TestConfig.class, PersistenceSpringConfig.class})
+public class VerifiedYieldContractCommodityDaoTest {
+	
+	@Autowired 
+	private PersistenceSpringConfig persistenceSpringConfig;
+	
+	private Integer growerContractYearId = 99999999;
+	private Integer contractId = 888888888;
+	private Integer cropYear = 2020;
+	private String verifiedYieldContractGuid;
+	private String declaredYieldContractGuid;
+	private String inventoryContractGuid;
+
+
+	@Before
+	public void prepareTests() throws NotFoundDaoException, DaoException{
+		delete();
+	}
+
+	@After 
+	public void cleanUp() throws NotFoundDaoException, DaoException{
+		delete();
+	}
+	
+	private void delete() throws NotFoundDaoException, DaoException{
```

Comment:
Delete verified yield contract commodity should be added to the cleanup in case the test fails